### ISSUE TITLE
fix methods with ref parameters

### ISSUE THREAD

**Issue by clement-brodu** [clement-brodu/PCTTools#6]
`ref` parameters (aka INPUT-OUTPUT in Openedge) are not visible in the json :

```csharp
public string GetMyPropertyWithInputOutput(int hello, ref string name, out string othername)
```

Openedge equivalent 

```abl
METHOD PUBLIC CHARACTER GetMyPropertyWithInputOutput (INPUT hello AS INTEGER, INPUT-OUTPUT name AS CHARACTER,  OUTPUT othername AS CHARACTER)
```

```json
        {
          "name": "GetMyPropertyWithInputOutput",
          "returnType": "CHARACTER",
          "parameters": [
            {
              "name": "hello",
              "type": "INTEGER"
            },
            {
              "name": "name",
              "type": "System.String"
            },
            {
              "isOut": true,
              "name": "othername",
              "type": "System.String"
            }
          ]
        },
```

@gquerret , any preference for the INPUT-OUTPUT ? 
```json
            {
              "isInputOutput": true, // true if input-output 
              "isOut": false, // true if output
              "name": "othername",
              "type": "System.String"
            }
```
or 

```json
            {
              "mode": "IO" // I for input or O for output or IO for input-output (default to "I")
              "name": "othername",
              "type": "System.String"
            }
```
or 

```json
            {
              "mode": "INPUT-OUTPUT" // explicit openedge keyword
              "name": "othername",
              "type": "System.String"
            }
```

**Comment by gquerret** [comment]
I'd go for `"mode"="IO"`.

**Comment by clement-brodu** [comment]
fixed in 1.0.3

schemaVersion is still `1` because it is not in production yet 😉
